### ISSUE TITLE
Handle CLOSED renewal before approval

### ISSUE THREAD

**Issue by anncatton** [icgc-argo/dac-api#370]
If a renewal is manually CLOSED before it is approved, it should be unlinked from the original application, so the original app can be renewed again if still within the renewable period. **This is a separate scenario from the batch job closure of renewals that have run out of time.**

[ ] - on CLOSED state change, remove the `renewalAppId` from the original application

**Comment by b-f-chan** [comment]
Behind feature-flag in production, to be tested properly in QA once feature is ready

**Comment by anncatton** [comment]
Cannot verify in production; will open new ticket should an issue arise.
